modules/combined/tests/generalized_plane_strain_tm_contact/generalized_plane_strain_tm_contact_field.i

```diff
@@ -2,7 +2,7 @@
   order = FIRST
```

Comment:
This test was renamed by removing the word 'field'. And I realived this test using the new formulation but without 'quadrature' option. Now this test was passed.